fedora_conf/data/load_order

```diff
@@ -49,3 +49,4 @@ dd327qr3670.xml
 fg464dn8891.xml
 pb873ty1662.xml
 qq613vj0238.xml
+bb007hx5508.xml
```

Comment:
Can we write a test so that we don't have to load more fixtures?  These are very hard to manage and they're slow to load.

Reply:
Sorry, that was cruft and has been removed.